README.md

```diff
@@ -375,6 +376,44 @@ var client = new AddinClient({...});
 client.hideWait();
 ```
 
+#### Sending custom events to the host page
+**Blackbaud Internal Only**
+An add-in can send a custom event to the host page, as long as the host supports the event type.
+Before sending an event to the host page, you must register for `init` and check the `supportedEventTypes`
+property to determine what event types the host page will handle.
+
+```js
+var supportedEventTypes;
+
+var client = new AddinClient({
+  callbacks: {
+    init: (args) => {
+      supportedEventTypes = args.supportedEventTypes;
+
+      args.ready({
+        showUI: true
+      });
+    }
+  }
+});
+
+// Before sending event, check to make sure the event is supported by the host page
+if (supportedEventTypes.includes('my-event-type')) {
+  // To send add-in events to the host, call the sendEvent method
+  client.sendEvent({
+    type: 'my-event-type',
+    context: { /* arbitrary context object to pass to host page */ }
```

Comment:
This context object isn't really "arbitrary", is it?  It'd be whatever shape is expected by the host page for the specific event being sent right?  So if/when this feature becomes something supported for 3rd-party scenarios, not only would we document supported event types on a per-host basis, but we'd also document the supported context for each event type.  So just wondering if it'd be better to just remove "arbitrary" from this comment (or clarify that this context shape is defined by the host page for the event being sent)

Reply:
You're right!  Yeah just removing `arbitrary` for now seems good to me.

And yes, per extension point we could document what the shape of the context object could be.  Similar to what we do for the extension points (Gift entry form and Gift Batch entry form) that support host events: https://developer.blackbaud.com/skyapi/docs/addins/concepts/extension-points#gift

Reply:
@Blackbaud-BenLambert Doc is now updated